packages/core/src/components/select/select.scss

```diff
@@ -17,6 +17,8 @@
   position: relative;
   height: auto;
   width: 100%;
+  box-shadow: inset 3px 3px 5px 6px var(--theme-color-shadow-3);
```

Comment:
`box-shadow` is not visible and check if you can use the prefinited shadow:

```css
box-shadow: var(--theme-inset-shadow-1);
```
https://ix.siemens.io/docs/theming/shadow